tests/functional/z_cluster/test_storagesystem.py

```diff
@@ -32,7 +33,10 @@ def test_storagesystem_not_present(self):
         storage_system = ocp.OCP(
             kind=constants.STORAGESYSTEM, namespace=config.ENV_DATA["cluster_namespace"]
         )
-        storage_system_data = storage_system.get()
+        try:
+            storage_system_data = storage_system.get()
```

Comment:
If storage system is completely removed from 4.19, why do we need a try/catch? Can't we simply remove it?